packages/monaco-graphql/src/utils.ts

```diff
@@ -11,13 +11,13 @@ import type {
   Diagnostic,
   CompletionItem as GraphQLCompletionItem,
 } from 'graphql-language-service-types';
+import type monaco from 'monaco-editor';
```

Comment:
it's available globally across the package now, so no need for this! i just need to add it to eslint globals